wagtail/wagtailadmin/forms.py

```diff
@@ -170,6 +169,12 @@ def clean(self):
         # New parent page given in form or parent of source, if parent_page is empty
         parent_page = cleaned_data.get('new_parent_page') or self.page.get_parent()
 
+        # check if user is allowed to create a page at given location.
+        if not parent_page.permissions_for_user(self.user).can_add_subpage():
+            self._errors['new_parent_page'] = self.error_class(
+                [_("You do not have permission to copy to page \"%s\"" % parent_page)]
```

Comment:
`parent_page` should be passed from outside of the `_(...)` call, as per the example at https://docs.djangoproject.com/en/1.11/topics/i18n/translation/#standard-translation